### DISCUSSION TITLE
Clone components and its children

### DISCUSSION THREAD

**Discussion by elisandroesp** [OFFLINE-GmbH/oc-boxes-support#115] [Ideas and feature requests]
Right now, when we duplicate a component, its children are not cloned.
That would be the default behaviour for cloning most components, otherwise you would use the 'add box' button to insert an empty one

**Comment by tobias-kuendig** [comment]
@elisandroesp Thank you for this report. This was due to a regression introduced in a previous update.

This should work now as expected in v3.5.12.

**Reply by elisandroesp** [reply]
Awesome! 
It's great to see your product evolving so quickly!
